code/go/0chain.net/smartcontract/dbs/event/blobber.go

```diff
@@ -122,14 +122,11 @@ func (edb *EventDb) addOrOverwriteBlobber(blobber Blobber) error {
 }
 
 func (bl *Blobber) exists(edb *EventDb) (bool, error) {
-	var count int64
-	result := edb.Get().
-		Model(&Blobber{}).
-		Where(&Blobber{BlobberID: bl.BlobberID}).
-		Count(&count)
+	var exists bool
+	result := edb.Get().Raw("select exists(select 1 from blobbers where blobber_id = ? limit 1) as ex", bl.BlobberID).Take(&exists)
```

Comment:
result := edb.Store.Get().Model(&Blobber).Where(&Blobber{BlobberID: bl.BlobberID}).Take(&blobber)

@samtholiya please change it to this